cs/TagsCloudVisualization/Spiral.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Drawing;
+
+namespace TagsCloudVisualization
+{
+    /// <summary>
+    /// Класс реализует  спираль Архимеда
+    /// </summary>
+    public class Spiral
+    {
+        private const double TwoPi = 2 * Math.PI;
+
+        private const double Step = 0.5;
+
+        private int countPoints;
+
+        private Point center;
+
+        public Spiral(Point center)
+        {
+            this.center = center;
+            countPoints = 0;
+        }
+
+        public Point CalcPointSpiral()
```

Comment:
Calc?
[Canadian Association of Lutheran Congregations?](https://en.wikipedia.org/wiki/Canadian_Association_of_Lutheran_Congregations)